package-lock.json

```diff
@@ -9,6 +9,7 @@
             "version": "0.0.0",
             "dependencies": {
                 "@gsap/react": "^2.1.0",
+                "prop-types": "^15.8.1",
```

Comment:
WHy we need this package?

Reply:
This package includes runtime type checking for React props and similar objects.

Reply:
done